src/senaite/core/api/dtime.py

```diff
@@ -139,7 +140,11 @@ def to_DT(dt):
         except (SyntaxError, IndexError):
             return None
     elif is_dt(dt):
-        return DateTime(dt.isoformat())
+        try:
+            # XXX Why do this instead of DateTime(dt)?
+            return DateTime(dt.isoformat())
```

Comment:
Not sure anymore, but I guess it was also related to the timezone, e.g. GMT vs. Etc/GMT or CET etc.